var/spack/repos/builtin/packages/arbor/package.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Arbor(CMakePackage):
+    """Arbor is a high-performance library for computational neuroscience
+    simulations."""
+
+    homepage = "https://github.com/arbor-sim/arbor/"
+    url      = "https://github.com/arbor-sim/arbor/archive/v0.2.tar.gz"
+
+    version('0.2', sha256='43c9181c03be5f3c9820b2b50592d7b41344f37e1200980119ad347eb7bcf4eb')
+
+    variant('vectorize', default=False,
+            description='Enable vectorization of computational kernels')
+    variant('gpu', default=False, description='Enable GPU support')
+    variant('mpi', default=False, description='Enable MPI support')
+    variant('python', default=False,
+            description='Enable Python frontend support')
+    variant('unwind', default=False,
+            description='Enable libunwind for pretty stack traces')
+
+    depends_on('cuda', when='+gpu')
+    depends_on('mpi', when='+mpi')
+    depends_on('libunwind', when='+unwind')
+
+    extends('python@3.6:', when='+python')
+    depends_on('py-mpi4py', when='+mpi+python', type=('build', 'run'))
+
+    depends_on('cmake@3.9:', type='build')
+    # mentioned in documentation but shouldn't be necessary when
+    # using the archive
+    # depends_on('git@2.0:', type='build')
+
+    # compiler dependencies
+    # depends_on(C++14)
+    # depends_on('gcc@6.1.0:', type='build')
+    # depends_on('llvm@4:', type='build')
+    # depends_on('clang-apple@9:', type='build')
+
+    # when building documentation
+    depends_on('py-sphinx', type='build')
```

Comment:
Are the docs built by default? If not, or if there's an option to disable the docs, I would remove this (or make it part of a variant). Sphinx is a very large dependency to add to a package.

Reply:
I don't think so, but I also don't use that package on my own -> I'd prefer to leave it in (at least we don't care about an additional sphinx dependency, as it will be build anyways), but I marked it with a comment, so that in the future people might remove it after they tested it.